### DISCUSSION TITLE
Want to contribute to space-ros

### DISCUSSION THREAD

**Discussion by bailaC** [space-ros/space-ros#129] [Q&A] [answered]
Dear community members, I would like to contribute in space-ros but not sure where to start with. Can someone guide/mentor me to help me understand the overall goal? I have good knowledge of computer science and some beginner knowledge in ROS2, trying to contribute and learn ros2_control. Thanks in advance.

**Comment by Bckempa** [comment, marked as answer]
Hi @bailaC, welcome and thank you for volunteering.

We're currently heads-down preparing the quarterly release that freezes this week (you can see our progress [here](https://github.com/orgs/space-ros/projects/3/views/6)) so we don't have a ton of obvious beginner tickets or bandwidth right this second, but we'll be planning the April release soon after that which would be a great time to jump in.

In the meantime, can you expand more on what you'd be interested in working on? I see you got a PR accepted into `ros2_control`, congratulations!
